core/org.osate.help/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.osate.help;singleton:=true
-Bundle-Version: 2.10.0.qualifier
+Bundle-Version: 2.10.1.qualifier
```

Comment:
This version must be 2.10.0 (even if a version error is displayed). It should match the release version.

Reply:
done in commit 201a300